plugin/gcp/network.py

```diff
@@ -0,0 +1,83 @@
+# #######
+# Copyright (c) 2014 GigaSpaces Technologies Ltd. All rights reserved
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#        http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+#    * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+#    * See the License for the specific language governing permissions and
+#    * limitations under the License.
+from plugin.gcp import utils
+from plugin.gcp.service import GoogleCloudPlatform
+from plugin.gcp.service import blocking
+
+
+class Network(GoogleCloudPlatform):
+    def __init__(self,
+                 config,
+                 logger,
+                 network):
```

Comment:
Similar to the other classes - let's pass the network directly only to the methods that require it